docs/get-started/alert-templates.md

```diff
@@ -0,0 +1,61 @@
+# PMM alert templates
+
+Alert templates provide a set of common events and expressions for alerting, serving as a foundation for creating alert rules.
+
+Percona Monitoring and Management (PMM) offers three categories of alert templates to enhance database performance monitoring:
+
+1. **Built-in templates**: templates that are available out-of-the-box with the PMM installation and are available to all PMM users.
+2. **Percona Platform templates**: additional templates available exclusively to Percona customers who [connect PMM to Percona Platform](../how-to/integrate-platform.md) using a Percona Account.
```

Comment:
```suggestion
2. **Percona Platform templates**: additional templates dynamically delivered to PMM if the instance [connected to Percona Platform](../how-to/integrate-platform.md) using a Percona Account.
```

It Percona Platfrom - it can be  "all user " templates, not customers only.